common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -714,54 +694,12 @@ def test_wiki_tab_first_time(self):
         )
         self.assertEqual(expected_article_name, course_wiki.article_name)
 
-    # TODO: TNL-6546: This whole function will be able to go away, replaced by test_course_home below.
-    def test_courseware_nav(self):
-        """
-        Navigate to a particular unit in the course.
-        """
-        # Navigate to the course page from the info page
-        self.course_info_page.visit()
-        self.tab_nav.go_to_tab('Course')
-
-        # Check that the course navigation appears correctly
-        EXPECTED_SECTIONS = {
-            'Test Section': ['Test Subsection'],
-            'Test Section 2': ['Test Subsection 2', 'Test Subsection 3']
-        }
-
-        actual_sections = self.courseware_page.nav.sections
-
-        for section, subsections in EXPECTED_SECTIONS.iteritems():
-            self.assertIn(section, actual_sections)
-            self.assertEqual(actual_sections[section], EXPECTED_SECTIONS[section])
-
-        # Navigate to a particular section
-        self.courseware_page.nav.go_to_section('Test Section', 'Test Subsection')
-
-        # Check the sequence items
-        EXPECTED_ITEMS = ['Test Problem 1', 'Test Problem 2', 'Test HTML']
-
-        actual_items = self.courseware_page.nav.sequence_items
-        self.assertEqual(len(actual_items), len(EXPECTED_ITEMS))
-        for expected in EXPECTED_ITEMS:
-            self.assertIn(expected, actual_items)
-
-        # Navigate to a particular section other than the default landing section.
-        self.courseware_page.nav.go_to_section('Test Section 2', 'Test Subsection 3')
-        self.assertTrue(self.courseware_page.nav.is_on_section('Test Section 2', 'Test Subsection 3'))
-
     def test_course_home_tab(self):
         """
         Navigate to the course home page using the tab.
         """
-        # TODO: TNL-6546: Use tab navigation and remove course_home_page.visit().
```

Comment:
Reminder that we aren't doing TNL-6546, which is removal of `course_outline_page`.  We are working with `unified_course_tab`, and we are not removing the flag, but are only updating its default.

Given that, these particular changes are probably fine and could have been done when the default for `course_outline_page` was changed.